utils/zerovec/src/ule/chars.rs

```diff
@@ -52,15 +52,15 @@ pub struct CharULE([u8; 4]);
 unsafe impl ULE for CharULE {
     #[inline]
     fn validate_byte_slice(bytes: &[u8]) -> Result<(), ZeroVecError> {
-        if bytes.len() % 4 != 0 {
+        if bytes.len() % 3 != 0 {
             return Err(ZeroVecError::length::<Self>(bytes.len()));
         }
         // Validate the bytes
-        for chunk in bytes.chunks_exact(4) {
+        for chunk in bytes.chunks_exact(3) {
             // TODO: Use slice::as_chunks() when stabilized
             #[allow(clippy::indexing_slicing)]
-            // TODO(#1668) Clippy exceptions need docs or fixing.i
-            let u = u32::from_le_bytes([chunk[0], chunk[1], chunk[2], chunk[3]]);
+            // TODO(#1668) Clippy exceptions need docs or fixing.ti
```

Comment:
Request: Address the TODO.

```suggestion
            // Won't panic because the chunks are always 3 bytes long
```